src/design-system/Tooltip.tsx

```diff
@@ -24,6 +24,7 @@ const TooltipContentsDiv = styled.div`
   border-radius: 5px;
   min-width: 20px;
   min-height: 20px;
+  max-width: 400px;
```

Comment:
This ensure very long tooltip text doesn't just create an extremely long one-line tooltip across the entire page.